cli.js

```diff
@@ -17,10 +18,35 @@ const source = `
 @tailwind utilities;
 `;
 
+/**
+ * @returns {String[]} A list of provided custom colors
+ */
+const getCustomColors = () => {
+	try {
+		const config = require(path.resolve('tailwind.config'));
+
+		if (
+			config &&
+			config.theme &&
+			config.theme.extend &&
+			config.theme.extend.colors
+		) {
+			return Object.keys(config.theme.extend.colors);
+		}
+	} catch (error) {
+		if (error.code !== 'MODULE_NOT_FOUND') {
+			throw error;
+		}
```

Comment:
Before `require()`ing a file, check for its existence first to avoid this error handling.

Reply:
I have the change I want to push up but I don't have permissions

Reply:
Feel free to fork this PR and open your own.